pretrained_models/README.md

```diff
@@ -0,0 +1,133 @@
+![image](resources/DeepCell_website_predict.png)
+
+2. This will take you to the image upload interface. The default model is `multiplex`. Before submitting your image, make sure you understand [the available models and data formatting requirements](#formatting-data-for-web-based-models). Upload your image by dragging it into the upload box, or by clicking and then browsing to find your image.  
+
+![image](resources/DeepCell_website_upload.png)
+
+3. Once your image has been successfully uploaded, click `Submit`, and the server will begin processing your data.  
+
+![image](resources/DeepCell_website_submit.png)
+
+4. Once complete, you can download the results, and then process additional images.  
+
+![image](resources/DeepCell_website_download.png)
+
+### Generating predictions with ImageJ
+
+The ImageJ plugin provides an easy interface to access our pre-trained deep learning models. Data is automatically uploaded to our server, processed, and then returned within ImageJ.
+
+Before getting started, make sure you understand [the available models and data formatting requirements](#formatting-data-for-pre-trained-models).
+
+#### Installation
+
+To install the ImageJ plugin, follow the [instructions](https://github.com/vanvalenlab/kiosk-imageJ-plugin#how-to-install).
+
+#### Generate predictions
+
+To generate predictions using the plugin, follow the [instructions](https://github.com/vanvalenlab/kiosk-imageJ-plugin#how-to-run-the-plugin).
+
+## Running pre-trained models locally
+
+Although cloud-based deployments are convenient and lower the barrier to entry for new labs, the degree of customization and throughput is limited compared to running the models locally. For users with large volumes of imaging data or specific requirements, we have also made our models available to be run locally or via a cluster. 
+
+### Jupyter notebook
+The [deepcell-tf application](https://github.com/vanvalenlab/deepcell-tf/tree/master/notebooks/applications) notebooks provide an easy interface to run our pre-trained models. Please follow the instructions in the [deepcell-tf ReadMe](https://github.com/vanvalenlab/deepcell-tf/blob/master/README.md) to set up the repository, then select the application notebook that corresponds best to your specific use case. 
+
+### Runnable docker image
+For users who want to integrate one of our models with an existing image analysis workflow, we created a runnable Docker image that can be directly called from the command line as part of a larger workflow. See the [ReadMe](https://github.com/vanvalenlab/deepcell-applications/blob/master/README.md) for instructions on getting started
+
+### Multiplex image analysis pipeline
+For users who are specifically interested in analyzing multiplexed image data, our collaborators at the Angelo Lab have set up their own analysis pipeline which uses the DeepCell ecosystem for cell segmentation. See their [ReadMe](https://github.com/angelolab/ark-analysis) for instructions on getting started
```

Comment:
White space, punctuation, and a personal all-caps preference for README.

```suggestion
### Jupyter notebook

The [deepcell-tf application](https://github.com/vanvalenlab/deepcell-tf/tree/master/notebooks/applications) notebooks provide an easy interface to run our pre-trained models. Please follow the instructions in the [deepcell-tf README](https://github.com/vanvalenlab/deepcell-tf/blob/master/README.md) to set up the repository, then select the application notebook that corresponds best to your specific use case. 

### Runnable docker image

For users who want to integrate one of our models with an existing image analysis workflow, we created a runnable Docker image that can be directly called from the command line as part of a larger workflow. See the [README](https://github.com/vanvalenlab/deepcell-applications/blob/master/README.md) for instructions on getting started.

### Multiplex image analysis pipeline

For users who are specifically interested in analyzing multiplexed image data, our collaborators at the Angelo Lab have set up their own analysis pipeline which uses the DeepCell ecosystem for cell segmentation. See their [README](https://github.com/angelolab/ark-analysis) for instructions on getting started.
```

Reply:
Do we need "Runnable docker image"? Should it just be "Docker Image"
Docker should probably be capitalized either way.

Reply:
Maybe executable docker image? The thing I was trying to communicate was the difference between the docker for deepcell-tf, which launches a jupyter notebook, and this docker, which is used as a command-line tool to directly run the model. Open to suggestions, or if we should just change it to docker.

Reply:
Agree on the capitalization.

Technically, it's almost the same docker image with a different CMD. The thing we want to highlight is that `deepcell` can explore and interact with data via Jupyter or as a code-driven programmable stage in a greater pipeline..